PolyEngine/Engine/Src/KeyBindings.hpp

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
+#if defined(_WIN32)
 #include <Windows.h>
 
 namespace Poly
 {
-	enum class eKey
-	{	
+	enum class eKey : uint8_t
```

Comment:
It's always better to cach an assertion with value 257 than have it overflow to 1 and go completely silent.